src/plugins/intel_gpu/src/graph/impls/ocl/kernels_cache.cpp

```diff
@@ -648,9 +666,24 @@ void kernels_cache::load(BinaryInputBuffer& ib) {
     size_t num_cached_binaries;
     ib >> num_cached_binaries;
     for (size_t i = 0; i < num_cached_binaries; ++i) {
+        bool is_zebin_binary = true;
+
         uint32_t id;
         ib >> id;
         ib >> precompiled_kernels[id];
+        ib >> is_zebin_binary;
+        if (!is_zebin_binary) {
+            // Legacy patchtoken path
+            std::string driver_version, current_driver_version;
+            ib >> driver_version;
+            current_driver_version  = downcast<ocl::ocl_device>(*_device).get_info().driver_version;
```

Comment:
nit: we already have `const auto& build_device = downcast<ocl::ocl_device>(*_device);` below.